integration/functions.sh

```diff
@@ -0,0 +1,189 @@
+# Functions for setting up Trillian integration tests
+
+if [ "${TMPDIR}" == "" ]; then
```

Comment:
nit: `if [[ -z "${TMPDIR}" ]]; then ...`

`[[` instead of `[` and `-z` for the empty check.